index.js

```diff
@@ -7,7 +7,9 @@ var binPath
 module.exports = function () {
   if (binPath) return binPath
 
-  if (process.platform === 'win32') {
+  if (process.env.PREFIX) {
+    binPath = process.env.PREFIX
+  } else if (process.platform === 'win32') {
```

Comment:
You can use [this package](https://github.com/jonschlinkert/is-windows) for detecting windows. It has some additional checks